app/src/app/framework/services/type.service.ts

```diff
@@ -62,7 +64,40 @@ export class TypeService<T = CmdbType> implements ApiService {
 
   public servicePrefix: string = 'types';
 
-  constructor(private api: ApiCallService) {
+  constructor(private api: ApiCallService, private userService: UserService) {
+  }
+
+  /**
+   * Returns acl read filter
+   *
+   * @private
+   */
+  private getAclReadFilter() {
```

Comment:
Why do you out-stock this function? If I see correctly it is only used in one place.

Reply:
This is used at 4 different places, getTypeList, getTypesByNameOrLabel, getUncategorizedTypes and getTypeListByCategory